inst/WORDLIST

```diff
@@ -56,7 +56,6 @@ Nfleet
 Nsexes
 Nsurveys
 OFL
-Oken
```

Comment:
I don't know what the wordlist does but I just wanted to confirm that this line should be removed since I see other last names in here that have been retained.